tests/tests_suite_test.go

```diff
@@ -44,6 +44,10 @@ var (
 )
 
 func getDir() string {
+	// if inside Docker container, don't return nothin'
```

Comment:
Is it worth specifying we're inside a container explicitly with an env var? For example, the minio project uses `DOCKERIMAGE=1` to indicate that it's running inside a container